spec/ics-009-loopback-client/README.md

```diff
@@ -0,0 +1,196 @@
+---
+ics: 9
+title: Loopback Client
+stage: draft
+category: IBC/TAO
+kind: instantiation
+author: Christopher Goes <cwgoes@tendermint.com>
+created: 2020-01-17
+modified: 2020-01-17
+requires: 2
+implements: 2
+---
+
+## Synopsis
+
+This specification describes a loop-back client, designed to be used for interaction over the IBC interface with modules present on the same ledger.
+
+### Motivation
+
+Loop-back clients may be useful in cases where the calling module does not have prior knowledge of where precisely the destination module lives and would like to use the uniform IBC message-passing interface.
```

Comment:
Maybe provide a more concrete use case? I understand the rationale, but Idk why would one use a lookback client in the first place

Reply:
Because the interface is standardised. Say that you want to use the same (asynchronous) logic for handling token transfers to/from other modules on the same chain and to/from other modules on remote chains - this makes that easy to do without any overhead costs.

Reply:
As a direct analogy, people use TCP/IP loopback clients all the time (`127.0.0.1`) for precisely this reason - so that they can use all the existing tooling which supports the protocol.

Reply:
I added a quick note that `127.0.0.1` is similar.